Engine/source/gui/controls/guiTextCtrl.cpp

```diff
@@ -188,7 +188,7 @@ void GuiTextCtrl::setText(const char *txt)
    if( !mProfile )
       return;
    
-   if (txt)
+   if (txt && txt != mText)
```

Comment:
Is that pointer comparison? Surely neither of these are `StringTableEntry`s so wouldn't a string comparison be more appropriate? And if so, it begs the question whether it's worth doing a comparison to avoid a copy :P.